test/integration/nodejs/testdata/cpu_hog.js

```diff
@@ -0,0 +1,14 @@
+const fib = (n) => {
```

Comment:
We should change the code example, as I commented above.

Reply:
OK